src/main/java/com/cloudant/client/org/lightcouch/CouchDbClientBase.java

```diff
@@ -77,85 +63,49 @@
     static final Log log = LogFactory.getLog(CouchDbClient.class);
 
     private URI baseURI;
-    final HttpClient httpClient;
-    final HttpHost host;
+
     private Gson gson;
 
-    final CookieStore cookies = new BasicCookieStore();
+    private Map<String, String> customHeaders;
+
+    private List<HttpConnectionRequestInterceptor> requestInterceptors;
+    private List<HttpConnectionResponseInterceptor> responseInterceptors;
 
     CouchDbClientBase() {
         this(new CouchDbConfig());
     }
 
     CouchDbClientBase(CouchDbConfig config) {
-
         final CouchDbProperties props = config.getProperties();
-        if (props.getAuthCookie() != null) {
-            setCookie(props);
-        }
-        this.httpClient = createHttpClient(props);
-        this.host = new HttpHost(props.getHost(), props.getPort(), props.getProtocol());
+
         this.gson = GsonHelper.initGson(new GsonBuilder()).create();
         final String path = props.getPath() != null ? props.getPath() : "";
-        this.baseURI = buildUri().scheme(props.getProtocol()).host(props.getHost()).port(props
-                .getPort()).path("/").path(path).build();
-
-        // for the authentication using previous session's cookie . do not get the cookie
-        if (props.getAuthCookie() == null) {
-            getCookie(props);
+        //Add username and password if authentication info exists

```

Comment:
Yes - I left a note on another userInfo comment pointing to HttpConnection's execute method that uses userInfo to create authentication.